geant4_vmc.sh

```diff
@@ -13,6 +13,8 @@ env:
   G4VMCINSTALL: "$GEANT4_VMC_ROOT"
 ---
 #!/bin/bash -e
+export LD_LIBRARY_PATH=${ALIBUILD_SYSTEM_VIEW:+${ALIBUILD_SYSTEM_VIEW}/lib:}${LD_LIBRARY_PATH}
```

Comment:
why is this change necessary here (and not in all recipes)? If we have a system view ... I would assume the library environment is handled outside?

Reply:
The problem comes from the fact that `rootcling`, which comes from our installation, has dependencies on things in a view (e.g. libz), and such a view should not be guaranteed to be in the LD_LIBRARY_PATH at build time. The reason why we do not want the view to be included at build time is that it could provide externals which conflict with the developer environment (typical case is freetype conflicting with various GUIs like emacs) and therefore you want to limit their presence to when you do `alienv enter`.